src/c_api/c_api_symbolic.cc

```diff
@@ -696,3 +696,21 @@ int MXSetCalibTableToQuantizedSymbol(SymbolHandle qsym_handle,
   *ret_qsym_handle = s;
   API_END_HANDLE_ERROR(delete s);
 }
+
+int MXGenBackendSubgraph(SymbolHandle sym_handle, const char *backend,
```

Comment:
By backend in this context do you mean the symbol that's a placeholder for the fused mkldnn call?

Reply:
What do you mean by a placeholder for the fused mkldnn call?
This API is intend for converting a symbol into a backend specific symbol. On quantization flow, we need do fusion at first, and then do quantization. So on python level, we need to get the backend specific symbol that all backend specific fusion applied, and pass it to quantization pass. That's why we need this API.

Reply:
I guess my point is that 'backend' is an overloaded term here, so to me it's confusing when you say you're converting a symbol to a backend specific symbol.

Am I understanding correctly that when you're fusing ops and calling this function the symbol that you begin with (sym_handle) is a symbol representing a graph of NNVM ops which use our default MXNet backend, and the symbol you're converting to represents a fused operator targeting an MKLDNN backend?

Reply:
Yes, your understanding is correct.